src/crypto/index.js

```diff
@@ -1295,6 +1296,25 @@ class IncomingRoomKeyRequestCancellation {
 }
 
 /**
+ * The result of a (successful) call to decryptEvent.
+ *
+ * @typedef {Object} EventDecryptionResult
+ *
+ * @property {Object} clearEvent The plaintext payload for the event
+ *     (typically containing <tt>type</tt> and <tt>content</tt> fields).
+ *
+ * @property {string?} senderCurve25519Key Key owned by the sender of this event.
```

Comment:
You need to start with a `?` not end with one. http://usejsdoc.org/tags-type.html